src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -200,10 +207,17 @@ internal void DependencyRegen(WorkspaceModel ws, bool forceCompute = false)
             {
                 foreach (PackageDependencyInfo packageDependencyInfo in packageDependencies)
                 {
-                    var targetInfo = pmExtension.PackageLoader.LocalPackages.Where(x => x.Name == packageDependencyInfo.Name).FirstOrDefault();
-                    if (targetInfo != null)
+                    try
+                    {
+                        var targetInfo = pmExtension.PackageLoader.LocalPackages.Where(x => x.Name == packageDependencyInfo.Name).FirstOrDefault();
+                        if (targetInfo != null)
+                        {
+                            packageDependencyInfo.Path = targetInfo.RootDirectory;
```

Comment:
could it cause problems for this to be left null? should it be to set to empty string or something? I don't actually know if it's null - just something to consider.

Reply:
it could be null

![imagen](https://user-images.githubusercontent.com/96534278/213791952-8decea76-cee1-408b-8ae3-3345c036fa0c.png)